pkg/queue/queue.go

```diff
@@ -152,7 +153,10 @@ func (q *Queue) Start() {
 			continue
 		}
 
-		itm := e.Value.(*item)
```

Comment:
Panic was implicit here right? Why the explicit call to panic? Linter?

Reply:
This change is for the linter only. Similar to the error rule, somebody should make the decision to handle this. An alternative solution is the `//nolint` comment.